web/src/components/LeftPane/LeftPane.module.css

```diff
@@ -0,0 +1,46 @@
+.leftpane {
+  background-color: var(--color-light);
+  box-shadow: 2px 0 5px var(--color-grey);
+  flex-direction: column;
+  align-items: center;
+  justify-content: start;
+  display: flex;
+  padding: 1rem;
+  width: 400px;
+  height: 100vh;
+}
+
+.navlist {
+  list-style: none;
+  margin: 60px;
```

Comment:
The 60px margins/spacing seem quite large you may want to make these values responsive